presqt/api_v1/serializers/target.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import reverse
+from rest_framework import serializers
+
+
+class TargetsSerializer(serializers.Serializer):
+    """
+    Serializer for GET operation on multiple Target objects.
```

Comment:
Nit-picky

I'm not sure if the HTTP method being used is relevant in this circumstance.
It's not about whether it is a GET operation, but that is in for multiple resources (collection)...??